### PR TITLE
[HOLD] Content driven homepage and CDK product pages

### PR DESCRIPTION
**Pull request by adelasofia**

### CODE REVIEW

**Comment by paulrobinson** [comment]
@ollyjshaw can you investigate why this PR build is not being triggered?

**Comment by ollyjshaw** [comment]
@paulrobinson It's strange. I've checked everything, even made a new job. It's nothing specific to @adelasofia though. My PR also didn't work.  Either the permissions are off our end (github collaborators), or the jenkins box is having issues. I'll check with the admins tomorrow.

**Comment by ollyjshaw** [comment]
@RubyGemTSL @adelasofia @paulrobinson. Build is now working. This PR is failing due to acceptance tests.

**Comment by paulrobinson** [comment]
The issue with the acceptance tests is that /products has changed, but the tests have not been updated. See: http://it-developers-pr.stage.redhat.com/pr/600/build/1315/products/

However, there's an issue with the acceptance tests not using the PR branch. @RubyGemTSL what's the status on that?

File: index.html.slim [hunk 1]
```diff
@@ -6,118 +6,94 @@ title: Red Hat Developers
...
-          a.slider-next(data-direction="next")
-            i.fa.fa-arrow-circle-right.fa-2x
+        li
+          a href='#{site.base_url}/products/cdk/get-started/' Need RHEL to test CDK?  Get it free today.
+        li
+          a href='#{site.base_url}/products/cdk/get-started/' Download and install the CDK; build your first Docker container app
+        li
+          a href='https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/7/container-development-kit-guide/container-development-kit-guide' CDK guide with recommended practices
+        li
+          a href='https://developerblog.redhat/com/tags/containers' Recent Docker container articles
```
Comment (paulrobinson) [inline index.html.slim]: developerblog.redhat/com -> developerblog.redhat.com

File: index.html.slim [hunk 2]
```diff
@@ -6,118 +6,94 @@ title: Red Hat Developers
...
+      h5.caps Accelerate Your Development
+      .row
+        .medium-12.small-24.columns
+          a.video-thumb(href='')
```
Comment (paulrobinson) [inline index.html.slim]: Missing target url

**Comment by paulrobinson** [comment]
Link to CDK download on /downloads should go to: http://it-developers-pr.stage.redhat.com/pr/600/build/73/products/cdk/download/

**Comment by RubyGemTSL** [comment]
@paulrobinson its on my todo list, but i'll pick that up now

**Comment by paulrobinson** [comment]
Thanks @RubyGemTSL. It's pretty urgent as it's causing a lot of PR failures. Got a link to the JIRA issue?

**Comment by RubyGemTSL** [comment]
@paulrobinson https://issues.jboss.org/browse/RHD-1159

**Comment by RubyGemTSL** [comment]
@redhat-developer-ci run the tests

**Comment by ollyjshaw** [comment]
retest this please

**Comment by RubyGemTSL** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by RubyGemTSL** [comment]
@adelasofia @paulrobinson @ollyjshaw - I have emailed with information on what Adela needs to do in order to get this passing.

**Comment by adelasofia** [comment]
@RubyGemTSL Thank you! I just pushed those fixes

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by paulrobinson** [comment]
I'm not seeing the CDK listed on http://it-developers-pr.stage.redhat.com/pr/600/build/150/products/

**Comment by paulrobinson** [comment]
I think this is the latest preview: http://it-developers-pr.stage.redhat.com/pr/600/build/159

The Details link seems to have the wrong build. I've created this issue: https://issues.jboss.org/browse/RHD-1215

**Comment by paulrobinson** [comment]
See: http://it-developers-pr.stage.redhat.com/pr/600/build/159/products/cdk/get-started-php/

Notice: "Great, you've selected", but no mention of 'what' is selected.

**Comment by adelasofia** [comment]
retest this please

**Comment by paulrobinson** [comment]
These changes are now merged into this branch: https://github.com/redhat-developer/developers.redhat.com/tree/cdk_beta which is now staged at: http://it-developers.stage.redhat.com/#!. Any future changes need to be made in a PR to that branch. I'll explain this properly next time we chat.

**Pull request closed without merging**